rmgpy/cantherm/pdepTest.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
```

Comment:
It seems like the actual operation being tested is inside the `setUp` method and not within any particular test itself. 

These also seem more like functional tests than unit tests, since you are running an entire module and checking for file output. Maybe this file should explicitly mention that it is a functional test, and it should be specified with the attribute `@attr('functional')` above the class line like in `testGenerateReactions.py`.